l10n_eu_oss/hooks.py

```diff
@@ -0,0 +1,57 @@
+# -*- encoding: utf-8 -*-
+# Copyright 2021 Rodrigo Bonilla <rodrigo.bonilla@factorlibre.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, SUPERUSER_ID, _
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+def post_init_hook(cr, registry):
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
```

Comment:
This shouldn't be a hook, but a migration script.

Reply:
Done, it's my first migrations, I hope it is correct. :)